gitbook/src/monitoring/creating-a-dashboard.md

```diff
@@ -0,0 +1,360 @@
+# Creating a dashboard
+
+You can set up monitoring for Erigon using Grafana and Prometheus. Erigon has built-in support for this monitoring stack with pre-configured dashboards and metrics collection. This guide will walk you through setting up a complete monitoring dashboard for Erigon using Prometheus and Grafana, leveraging the built-in monitoring tools provided in the Erigon codebase.
+
+The monitoring setup leverages Erigon's built-in metrics system and provides comprehensive visibility into node performance, storage usage, and network activity. The pre-configured dashboards are actively maintained and include metrics for both execution and consensus layer operations when running with Caplin.
+
+## Prerequisites
+
+- Docker and Docker Compose installed
+- Erigon node running
+- Basic understanding of Prometheus and Grafana
+
+## Step 1: Enable Metrics in Erigon
+
+First, you need to enable metrics collection in your Erigon instance by adding the `--metrics` flag.
+
+```bash
+./erigon --metrics --datadir=/your/data/dir
+```
+
+If you need to specify a custom metrics address, use `--metrics.addr`.
+
+## Step 2: Configure Prometheus Targets
+
+Add your Erigon hosts to the Prometheus configuration file:
+
+1. Copy the default configuration: `./cmd/prometheus/prometheus.yml`
+2. Edit the file to include your Erigon instance endpoints
+3. Save the modified configuration file
+
+## Step 3: Launch Monitoring Stack
+
+Erigon provides a simple Docker Compose setup for the monitoring stack:
+
+```bash
+docker compose up -d prometheus grafana
+```
+
+Alternatively, you can use the make target:
+
+```bash
+make prometheus
+```
+
+## Step 4: Access Grafana Dashboard
+
+Once the containers are running, access Grafana at [localhost:3000](http://localhost:3000).
+
+**Default credentials:** `admin/admin`
+
+## Step 5: Pre-configured Dashboards
+
+Erigon comes with comprehensive pre-built dashboards that monitor various aspects of the node:
+
+### Main Dashboard Features
+The `erigon_internals.json` dashboard includes panels for:
```

Comment:
`erigon_internals.json` it's very complex low-level dashboard for devs.
`erigon.json` is higher-level board for Users.